actors/miner/tests/extend_sector_expiration_test.rs

```diff
@@ -1,13 +1,26 @@
+use fil_actor_market::DealSpaces;
 use fil_actor_miner::{
     power_for_sector, seal_proof_sector_maximum_lifetime, ExpirationExtension,
-    ExtendSectorExpirationParams, PoStPartition, SectorOnChainInfo, State,
+    ExpirationExtension2, ExtendSectorExpiration2Params, ExtendSectorExpirationParams,
+    PoStPartition, SectorClaim, SectorOnChainInfo, State,
 };
+use fil_actor_verifreg::Claim as FILPlusClaim;
```

Comment:
Can you use `ext` instead, and then remove the cargo dependency on fil_actor_verifreg?